docs-2.0/nebula-studio/quick-start/st-ug-console.md

```diff
@@ -0,0 +1,25 @@
+# Console
+
+Studio console interface as shown in following.
```

Comment:
the predicate is missing. xxx is as shown in the following figure.
It is better to say `as shown in the following xxx`, or `xx is shown as follows`